plugins/chainlink.prebuilt.Dockerfile

```diff
@@ -0,0 +1,17 @@
+##
+# Takes Chainlink core as a base image and layers in private plugins.
+##
+ARG BASE_IMAGE=public.ecr.aws/chainlink/chainlink:v2.23.0-plugins
+
+##
+# Final image
+##
+FROM ${BASE_IMAGE} AS final
+# This directory should only contain the plugin binaries.
+ARG LOCAL_PLUGIN_DIR=./plugins
+ARG LOCAL_LIB_DIR=./lib
```

Comment:
Can we plz standardise this to `$PKG_PATH/bin` and `$PKG_PATH/lib`, which is close to Filesystem Hierarchy Standard (FHS) Nix uses. Then a user can just produce a build in a path, and use that path as PKG_PATH arg to use in Dockerfile build.

Ideally, user just provides the PKG_PATH so don't need to add both bin/lib paths individually as here: https://github.com/smartcontractkit/chainlink-ton/pull/7/files#diff-6220dea5ae8acc1de6b46062027457942b4b91dc69755af0ab319bf3116f8a7cR38

Reply:
Like this: 7c6610e863 ?